src/json/function.rs

```diff
@@ -0,0 +1,67 @@
+use json::json::{Json};
```

Comment:
Perhaps this module could just be placed at the top level? It seems a little odd to have a module just for some functions.